### DISCUSSION TITLE
Is  there a way to enable context-aware auto completions?

### DISCUSSION THREAD

**Discussion by rochacbruno** [helix-editor/helix#8169] [Q&A] [answered]
Hi, I am migrating from neovim to helix and it is amazing, thanks!

I write  a lot of text (not code), on notes, markdown, documentation etc.

Using previous editors like micro-editor, neovim, emacs, there was context-aware completion, for example, if I have the text:

```
Bruce got a brave bracelet because he is the bravest brazilian in the bratslav
```

In the previous paragraph and then I start writing a new one.

```
And then br(C-x)
                     [uce]
                     [ave]
                     [acelet]
                     [avest]
                     [azilian]
                     [atslav]
```

The auto complete would suggest tokens found in the current buffer (or even open buffers, or even project files, regardless of a language server?

Is that something that I could activate on the lsp?

Thanks

**Comment by the-mikedavis** [comment, marked as answer]
This isn't implemented yet, see https://github.com/helix-editor/helix/issues/1063

**Comment by rochacbruno** [comment]
I was able to add common word completions using https://github.com/estin/simple-completion-language-server
